pkg/kds/client/sink.go

```diff
@@ -78,10 +78,10 @@ func (s *kdsSink) Start(stop <-chan struct{}) (errs error) {
 		if err != nil {
 			return errors.Wrap(err, "failed to receive a discovery response")
 		}
-		s.log.Info("DiscoveryResponse received", "type", rs.GetItemType())
+		s.log.Info("DiscoveryResponse received", "response", rs)
```

Comment:
Does it print full response? We are syncing secrets here. I don't want to log secret here